From 0f917eec8d234b9369d66a11f370f2bb40e46717 Mon Sep 17 00:00:00 2001
From: Kshitij Kulshreshtha <kshitij@math.hu-berlin.de>
Date: Thu, 25 Mar 2010 00:12:49 +0100
Subject: [PATCH] add one more device

found to work in the thread on linux-media@vger
---
 rtl2832u.c |    1 +
 rtl2832u.h |    2 ++
 2 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/rtl2832u.c b/rtl2832u.c
index 6a1a4ce..d9dd9af 100644
--- a/rtl2832u.c
+++ b/rtl2832u.c
@@ -84,6 +84,7 @@ static struct usb_device_id rtl2832u_usb_table [] = {
 	{ USB_DEVICE(USB_VID_KWORLD_1ST, USB_PID_KWORLD_WARM_3) },
 	{ USB_DEVICE(USB_VID_KWORLD_1ST, USB_PID_KWORLD_WARM_8) },
 	{ USB_DEVICE(USB_VID_GTEK, USB_PID_GTEK_WARM) },	
+	{ USB_DEVICE(USB_VID_LEADTEK, USB_PID_LEADTEK_WARM) },	
 	{ 0 },
 };
 
diff --git a/rtl2832u.h b/rtl2832u.h
index ca6700a..04034dc 100644
--- a/rtl2832u.h
+++ b/rtl2832u.h
@@ -39,6 +39,8 @@
 #define	USB_PID_GTEK_WARM			0x0837
 
 
+#define USB_VID_LEADTEK					0x0413
+#define USB_PID_LEADTEK_WARM			0x6a03
 
 
 
-- 
1.7.0.3

